### ISSUE TITLE
DHCP make me Crazy！！！！

### ISSUE THREAD

**Issue by Unstoppablesss** [YACReader/yacreader#384]
I am a frequent traveler and often need to access comics stored on my computer using my phone. Every time I update my Wi-Fi or go to a new location, I have to reconfigure my IPv4 address and host settings. 😫

At home, I can easily set a static IP address and have control over the router, so there are no issues. However, when I'm in other places like hotels, it becomes a hassle to set up the IP address again. Moreover, in hotels, I cannot fix my IPv4 address, which makes it even worse as I never know when the IP address will change. This means I have to go through the setup process again. OMG

It would be great if YACReader could automatically generate a server address based on the IP address, similar to how devices on a local network in macOS can be accessed using something like FLMBP.local. It would be even better if I could customize the name of this server address. 🙏

**Comment by Unstoppablesss** [comment]
Actually I find I can scan the QR code. It is great but I will say If I can access my comic which was already pin in the menu without any thinking, just tap it. You know, apple experience!!!!

**Comment by luisangelsm** [comment]
zero-config networking is something I would like to add, but getting a universal solution that works on all platforms without extra software dependencies is hard to achieve or maybe impossible. PRs are welcome.

BUT, one thing you can do is to use your mac computer name instead of the IP.

**Comment by Unstoppablesss** [comment]
> BUT, one thing you can do is to use your mac computer name instead of the IP.

Yes, That's the feature I want to use. But currently it only shows my ip. Do you know how to change my ip to my computer name? I will be very appreciate for that!
